srcpkgs/godot/template

```diff
@@ -29,6 +29,13 @@ post_extract() {
 
 pre_build() {
 	export CXXFLAGS=" $CXXFLAGS "
+	# work around missing -fstack-clash-protection on clang
+	cat > ./clang++-wrapper << "EOF"
```

Comment:
What is the reason behind using clang instead of gcc?

Reply:
package was already using clang, and from the godot wiki it looks like it's the best supported choice. i can try building for gcc though